### PR TITLE
add paths and extensions to make things fool proof

### PR DESCRIPTION
**Pull request by beckermr**
I've added some paths and fits extensions to make things a bit more explicit.

@esheldon for viz

### CODE REVIEW

File: desmeds/coaddinfo.py [hunk 1]
```diff
@@ -174,6 +174,15 @@ def _add_full_paths(self, info):
             info['filename'].replace('.fits','_psfcat.psf'),
         )
 
+        info['image_ext'] = 'sci'
+
+        info['weight_path'] = info['image_path']
+        info['weight_ext'] = 'wgt'
```
Comment (beckermr) [inline desmeds/coaddinfo.py]: One could easily prefer `wgt` instead of `weight`. IDK

File: desmeds/coaddinfo.py [hunk 2]
```diff
@@ -174,6 +174,15 @@ def _add_full_paths(self, info):
             info['filename'].replace('.fits','_psfcat.psf'),
         )
 
+        info['image_ext'] = 'sci'
```
Comment (esheldon) [inline desmeds/coaddinfo.py]: the extensions are set in the config file

File: desmeds/coaddinfo.py [hunk 3]
```diff
@@ -174,6 +174,15 @@ def _add_full_paths(self, info):
...
+
+        info['weight_path'] = info['image_path']
+        info['weight_ext'] = 'wgt'
+
+        info['bmask_path'] = info['image_path']
+        info['bmask_ext'] = 'msk'
+
+        info['seg_ext'] = 'sci'
```
Comment (esheldon) [inline desmeds/coaddinfo.py]: these are also set in the config file

**Comment by beckermr** [comment]
Right. I think we were setting these when we wanted things to be more general. IDK if we need that any more but no reason to change it now. I am closing this and will add this info myself elsewhere.

**Comment by esheldon** [comment]
Yes, this code is used in production, I'm wary of making major changes like this.

**Pull request closed without merging**